src/AppEntry.js

```diff
@@ -4,19 +4,25 @@ import { Provider } from 'react-redux';
 import registry, { RegistryContext } from './utilities/store';
 import App from './App';
 import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/helpers';
+import { IntlProvider } from 'react-intl';
+import messages from './locales/data.json';
```

Comment:
should be referencing the locales outside src to avoid extra step of updating another data.json

Reply:
We should not be referencing resources outside of the `src` folder. Ideally, we would only have one `locales` folder and it would be in the `src` directory.

Reply:
I guess the location should be in src but yeah shouldn't there be only one data.json which should be referenced everywhere and it should be the one generated from createDataJson which has the language objects

Reply:
There is also IntlProvider and IntlHelper from frontend-components/translations which does the processing of messages[language] with cases where the translation is not available